bcap/pkg/graphs/resource_models/Archaeological Site.json

```diff
@@ -3136,28 +3136,6 @@
                     "visible": true,
                     "widget_id": "10000000-0000-0000-0000-000000000004"
                 },
-                {
```

Comment:
@braydencstratusadv - Can you confirm that this node is still available after this change? Not sure if this is now gone because of a rebase / buggy resource model designer, or if this is an inadvertent side effect.